protocol/x/clob/keeper/liquidations_test.go

```diff
@@ -2018,6 +2018,9 @@ func TestPlacePerpetualLiquidation_Deleveraging(t *testing.T) {
 			}
 
 			if tc.expectedFilledSize == 0 {
+				mockIndexerEventManager.On("AddTxnEvent",
+					mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything,
```

Comment:
nit: Can the actual deleveraging event be derived here?

Reply:
Bankruptcy price isn't exposed by the API. I could add it to the test params, but that would require some more changes. Plus, the exact deleveraging event is being tested in multiple other e2e tests. So, I think it's reasonable to omit here.

Reply:
Can you add comment with the above details?

Reply:
done